### DISCUSSION TITLE
5.5.9 filter properties feature

### DISCUSSION THREAD

**Discussion by RomanKubiak** [RomanKubiak/ctrlr#119] [Releases]
This might be useful for all you panel builders you can now filter properties by a wildcard typed above the property pane. To clear the filter just type an empty one or press ESC. It was something i needed during development as the number of properties has grown to vast proportions. Number of bugfixes too obviously. I'm trying to write a short changelog with every release, there will be Windows and MAC releases as soon as i get access to some working hardware.

**Comment by Ted-juh** [comment]
Good addition. 

I don't know if there is any demand for besides the fact that I would like to see that option. Bit iss it doable/ possible to separate the input for X/Y/W/H of a component?

**Reply by shooking** [reply]
Can you elaborate what you mean? If I want to modify the position and size of components I do it from one of the view modulators OR via Lua. I keep meaning to make a video. But in effect you can program a panel layout but throwing a series of modulators on the screen - then in the Lua console window grab the handle for it (panel:getMod etc) get and use the XML info to determine which property yo get and then set it.

Just be careful to setProperty to the correct base - mod for the modulators stuff and comp for the component stuff.

Sorry of this isn't what you are driving at - or if I haven't explained myself properly.

**Reply by Ted-juh** [reply]
I have absolutely no clue what you are trying to explain, lol. But don't get me wrong, I should have been more clear.

I'm talking about the "ComponentRectangle/ Component Size and Position" properties under "Component Generic" settings in the Ctrlr Editor sidebar. 

![pic 1](https://user-images.githubusercontent.com/71118185/103955169-0518bd00-5146-11eb-8abe-ddf99a0382fa.jpg)

I wondered if it could be more like this:

![pic 2](https://user-images.githubusercontent.com/71118185/103955401-79ebf700-5146-11eb-8ec6-3bd8d53c53f3.jpg)
